package.json

```diff
@@ -123,6 +126,15 @@
       "scan": {
         "description": "[Scan]         Scan to get detailed security testing results: Visit https://saf.mitre.org/#/validate to explore and run inspec profiles"
       },
+      "supplement": {
+        "description": "[Supplement]   Supplement (ex. read or modify) elements that provide contextual information in the Heimdall Data Format results JSON file such as `passthrough` or `target`"
+      },
+      "supplement:passthrough": {
```

Comment:
Are these colon separated because we end up with three commands? What I mean is, the idea is `saf 1. supplement 2. passthrough/target 3. read/write flags and stuff`?

Reply:
Yes, the colons are syntax used by oclif to differentiate the subcommands.  In recent releases when actually calling the subcommand you can now use spaces, but that qol improvement hasn't propagated everywhere like in this section within the package.json as far as I can tell.

Reply:
Ah, but I see that you can do all three commands with spaces instead of colons based on what you wrote in the examples. Sounds good